src/dndbeyond/content-scripts/character.js

```diff
@@ -171,6 +171,12 @@ function rollAbilityOrSavingThrow(paneClass, rollType) {
             roll_properties["modifier"] = modifier;
         }
     }
+    // Wizard - War Magic - Saving Throw Bonus
+    if (character.hasClassFeature("Durable Magic") && character.getSetting("wizard-durable-magic", false) &&
+        rollType == "saving-throw") {
+        modifier = parseInt(modifier) + 2;
+        roll_properties["modifier"] = modifier;
```

Comment:
The modifier has to be a string and needs to start with `+` since it gets appended to the formula, that's what this line (from other similar places) is for : 
```js
            modifier = modifier >= 0 ? `+${modifier}` : `${modifier}`;
```